asapdiscovery-data/asapdiscovery/data/fitness.py

```diff
@@ -140,35 +148,48 @@ def parse_fitness_json(target: TargetTags) -> pd.DataFrame:
             f"Specified target is not valid, must be one of: {TargetTags.get_values()}"
         )
 
-    if target not in ("SARS-CoV-2-Mpro", "SARS-CoV-2-Mac1"):
+    if target not in ("SARS-CoV-2-Mpro", "SARS-CoV-2-Mac1", "ZIKV-NS2B-NS3pro"):
         raise NotImplementedError(
             f"Fitness data not yet available for {target}. Add to metadata if/when available."
         )
 
-    with open(SARS_CoV_2_fitness_data) as f:
-        data = json.load(f)
-    data = data["data"]
-    fitness_scores_bloom = pd.DataFrame(data)
-    # now get the target-specific entries.
-    fitness_scores_bloom = fitness_scores_bloom[
-        fitness_scores_bloom["gene"] == _TARGET_TO_GENE[target]
-    ]
-    if target == "SARS-CoV-2-Mac1":
-        # need to subselect from nsp3 multidomain to get just Mac1. See https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7113668/
-        fitness_scores_bloom = fitness_scores_bloom[
-            fitness_scores_bloom["site"].between(209, 372)
-        ]
-        fitness_scores_bloom[
-            "site"
-        ] -= 208  # reindex to set residue numbers to correct values
-    elif target == "SARS-CoV-2-Mpro":
-        # simpler; can just query the correct gene in the JSON.
+    ## START TODO
```

Comment:
We can abstract this, but can push down road for now.

Reply:
agreed, we can probably loop this into https://github.com/choderalab/asapdiscovery/issues/649